### ISSUE TITLE
MM Machines and Mekanism Universal Cables Not Interfacing

### ISSUE THREAD

**Issue by PsyberMind** [DarkPacks/SevTech-Ages#2285]
## Issue / Bug
The Universal Cables from Mekanism will not interface with Modular Machinery Power Inputs
However, the Mechanical Pipes and Logistical Pipes will interface with the fluid/item ports respectively

## Expected Behavior
When connecting power resources to my machines, I would have expected that the universal cables would draw power into the Machines, since the Mechanical/Logistical Pipes do connect to their resepctive in/outputs

## Possible Solution
Add Support for Power Transfer from Mekanism to MM

## Steps to Reproduce (for bugs)
(twitch.tv/PsyberMind - Latest VOD, Or if I'm live, ask for a demo - I'll get a time code for the issue as soon as my broadcast is done)

1. Build Modular Machinery Machine
2.Connect Mekanism Basic Universal Cable to Power Input
3. Connect Cable to Power Source (In my case, RFT Power Cell)
4. Observe as Power does NOT transfer to the Machine

## Context

Basically just trying to power my base in a non-hazardous way, without cables hanging all over the place

## Client Information

* Modpack Version: 3.0.6
* Java Version: 1.8
* Launcher Used: Twitch App Ported into MultiMC
* Memory Allocated: 8GB
* Server/LAN/Single Player: SSP
* Resourcepack Enabled?: Soartex Fanver
* Optifine Installed?: Yes, No Shaders
* Added Mods - Twitch Chat Integration (BetterTwitchChat) (ChatTweaks) - Client Side Only

**Comment by artdude543** [comment]
This is resolved in 3.0.7 which is now live! 😄
